en/asgardeo/docs/apis/restapis/actions.yaml

```diff
@@ -182,6 +196,9 @@ paths:
           schema:
             enum:
               - preIssueAccessToken
+              - preUpdatePassword
+              - preUpdateProfile
+              - preRegistration
```

Comment:
AFAIR we discussed to add these when the respective action type is supported by the product.

Reply:
I added back as we have incorporated a validation for not implemented types.